README.md

```diff
@@ -20,3 +20,98 @@ cli-kit new myproject
 
 You can see example usage [here](https://github.com/Shopify/cli-kit-example). This app is similar to
 the one that the generator generates.
+
+## Starting a New Project
+To begin creating your first `cli-kit` application, run:
+```bash
+cli-kit new myproject
+```
+Where `myproject` is the name of the application you wish to create.  Then, you will be prompted to
+select how the project consumes `cli-kit` and `cli-ui`.  The available options are:
+- Vendor (faster execution, more difficult to update dependencies)
+- Bundler (slower execution, easier dependency management)
+
+You're now ready to write your very first `cli-kit` application!
+
+## How do `cli-kit` Applications Work?
+The executable for your `cli-kit` app is stored in the "exe" directory.  To execute the app, simply
+run:
+```bash
+./exe/myproject
+```
+
+### Folder Structure
+* `/exe/` - Location of the executables for your application.
+* `/lib/` - Location of the resources for your app (modules, classes, helpers, etc).
+    * `myproject.rb` - The main file for your application.
+    * `myproject/` - Stores the various commands/entry points.
+        * `entry_point.rb` - Is executed when the application is launched, and handles commands.
```

Comment:
This is the file that is first called from the executable. It handles loading, and ...